File: Part8/Go/FunctionConcepts/FunctionConcepts_6.go
// Example with three default parameters
// Go does not support default parameters, this shows limitation
package main

import "fmt"

func threeDefaultParameters ( FirstParameter_Par int, SecondParameter_Par int, ThirdParameter_Par int ) {
    fmt.Printf ( "Parameters: %d %d %d\n", FirstParameter_Par, SecondParameter_Par, ThirdParameter_Par )
}

func main() {
    threeDefaultParameters ( 20, 25, 30 )
    // Go does not support default parameters, must provide all arguments
    threeDefaultParameters ( 5, 10, 15 )
}

File: Part8/Rust/FunctionConcepts/FunctionConcepts_6.rs
// Example with three default parameters
// Rust does not support default parameters, this shows limitation
fn three_default_parameters ( first_parameter_par: i32, second_parameter_par: i32, third_parameter_par: i32 )
{
    println! ( "Parameters: {} {} {}", first_parameter_par, second_parameter_par, third_parameter_par );
}

fn main ()
{
    three_default_parameters ( 20, 25, 30 );
    // Rust does not support default parameters, must provide all arguments
    three_default_parameters ( 5, 10, 15 );
}
